sdk/cosmos/azure-cosmos/src/main/java/module-info.java

```diff
@@ -20,7 +20,6 @@
     requires reactor.netty.core;
     requires reactor.netty.http;
     requires com.codahale.metrics;
-    requires com.fasterxml.jackson.module.afterburner;
```

Comment:
is this a breaking change?
cc: @lmolkova / @srnagar

Reply:
Removing `requires` is not a breaking change but removing `requires transitive` is a breaking change. In this case, we are just removing `requires`, so, this is okay.